v2/options.go

```diff
@@ -31,24 +31,18 @@ type ConnectionLostHandler func(*Options, *mqtt.ClientOptions, error)
 
 // Options stores configuration of the MQTT connection.
 type Options struct {
-	KeyPath                  string
-	CertPath                 string
-	CaPath                   string
-	ClientID                 string
-	Region                   string
-	BaseReconnectTime        time.Duration
-	MaximumReconnectTime     time.Duration
```

Comment:
Seems better not to remove and pass the value to `paho.mqtt.ClientOptions`.
https://github.com/eclipse/paho.mqtt.golang/blob/93c8a06e1e6dfeb17dd18a44e3e2f4e1db778237/options.go#L69

default value in `paho.mqtt.ClientOptions` is quite large.
```go
		MaxReconnectInterval:    10 * time.Minute,
```
https://github.com/eclipse/paho.mqtt.golang/blob/93c8a06e1e6dfeb17dd18a44e3e2f4e1db778237/options.go#L108